todoComments/todoComments.md

```diff
@@ -0,0 +1,185 @@
+| 39 | isValidGPSInput currently pops up an alert so not doing it here, may change so leaving code commented out. | src/client/app/components/groups/CreateGroupModalComponent.tsx | Unsure | - | isValidGPSInput currently returns the showErrorNotification, so the showErrorNotification was commented out to avoid having 2 of the same error messages being returned. |
+| 40 | We should probably go all to modal or popups for messages. | src/client/app/components/groups/EditGroupModalComponent.tsx | Related To An Issue | [Github issue #1202](https://github.com/OpenEnergyDashboard/OED/issues/1202) | Issue 1202 is about potentially creating a single modal in the app and dynamically swapping content. may affect this TODO |
+| 41 | isValidGPSInput currently pops up an alert so not doing it here, may change so leaving code commented out. | src/client/app/components/groups/EditGroupModalComponent.tsx | Unsure | - | isValidGPSInput currently returns the showErrorNotification, so the showErrorNotification was commented out to avoid having 2 of the same error messages being returned. |
+| 42 | This component still displays a dropdown arrow, even though a user cannot use the dropdown | src/client/app/components/groups/EditGroupModalComponent.tsx | Needs Issue Created | - | This formgroup has an input field, with a 'select' type. This shows a dropdown arrow, but the user cannot use it (not sure if the user shouldn't use it or if it's broken and the user can't use it because of that). |
+| 43 | The following line should work but does not (it does for meters). The Redux state has the name of hidden groups but it should not. A quick fix is to check if the name is null. This occurs because the group tree code is passing groups even if not displayable. | src/client/app/components/groups/EditGroupModalComponent.tsx | Needs Issue Created | - | - |
+| 44 | These types of plotly containers expect a lot of passed values and it gives a TS error. Given we plan to  replace this with the react hooks version and it does not seem to cause any issues, this TS error is being suppressed for now. | src/client/app/components/maps/MapCalibrationComponent.tsx | No Issue Needed | - | Typescript is causing an issue because of a lot of values being passed, react hook versions planned to be used, typescript error currently suppressed. I don't believe an issue is needed as it'll be fixed when the react hook versions are used. |
+| 45 | Maybe should do as a single popup? | src/client/app/components/meters/CreateMeterModalComponent.tsx | Unsure | - | TODO doesn't reference exactly what needs to be done as a single pop up, I'm unable to understand what it's referring to. |
```

Comment:
This refers to the fact that the checks are done separately so the messages are popped up by each one. Ideally, the tests would return the message and then the component would display when desired. These checks are used in mulitple places so requires a number of changes to do this.